src/org/usfirst/frc/team467/robot/Drive.java

```diff
@@ -283,9 +283,22 @@ private double ticksToFeet(double ticks) {
 		return feet;
 	}
 
+	/**
+	 * Sets the ramp time based on the elevator height if driving straight or about to drive straight,
+	 * or sets the ramp time to the minimum if turning in place or stopped.
+	 * 
+	 * @param elevatorHeight
+	 */
 	public void setRamp(int elevatorHeight) {
-		double heightPercent = (double) (RobotMap.ELEVATOR_BOTTOM_TICKS - elevatorHeight) / (RobotMap.ELEVATOR_BOTTOM_TICKS - RobotMap.ELEVATOR_TOP_TICKS);
-		double ramp = MathUtils.weightedAverage(RobotMap.ELEVATOR_LOW_DRIVE_RAMP_TIME, RobotMap.ELEVATOR_HIGH_DRIVE_RAMP_TIME, heightPercent);
+		double ramp;
+		LOGGER.debug("Speed left=" + left.sensorSpeed() + " right=" + right.sensorSpeed());
```

Comment:
You shouldn't grab the speed sensor values more than once - it's probably creating extra CAN bus traffic. If you really need this debug log message then you should get the values once before logging.